doc/mount.glusterfs.8

```diff
@@ -104,9 +104,10 @@ Set the subdirectory mount option [default: NULL, ie, no subdirectory mount]
 .TP
 \fBbackup\-volfile\-servers=\fRSERVERLIST
 Provide list of backup volfile servers in the following format [default: None]
+In SERVERLIST, if ':' is surrounded by '[' and ']', it will be not recognized as a separator.
 
-\fB$ mount \-t glusterfs \-obackup\-volfile\-servers=<server2>:\fR
-\fB       <server3>:...:<serverN> <server1>:/<volname> <mount_point>\fR
+\fB$ mount \-t glusterfs \-obackup\-volfile\-servers=[<server2>]:\fR
+\fB       [<server3>]:...:[<serverN>] <server1>:/<volname> <mount_point>\fR
```

Comment:
We should specify that the `[]` are optional and only required for ipv6 addresses that contain embedded `:`.